.github/workflows/release.yaml

```diff
@@ -16,4 +22,38 @@ jobs:
       - uses: katyo/publish-crates@v1
         with:
             registry-token: ${{ secrets.CARGO_REGISTRY_TOKEN }}
-  # add publish docker image
\ No newline at end of file
+  # add publish docker image
+  release_vscode_ext:
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-18.04
+            extension: so
+            target: x86_64-unknown-linux-gnu
+            code-target: linux-x64
+          - os: ubuntu-18.04
+            extension: so
+            target: aarch64-unknown-linux-gnu
+            code-target: linux-arm64
+          - os: macos-11
+            extension: dylib
+            target: x86_64-apple-darwin
+            code-target: darwin-x64
+          - os: macos-11
+            extension: dylib
+            target: aarch64-apple-darwin
+            code-target: darwin-arm64    
+    runs-on: ${{matrix.os}}
+    steps:  
+    - uses: actions/checkout@v2
+    - uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: nightly
+    - run: npm install -g vsce typescript esbuild
+    - run: rustup target add ${{matrix.target}}
```

Comment:
you can specify it in the toolchain action:
https://github.com/marketplace/actions/rust-toolchain